### PR TITLE
Refactor template - create_*.py

### PR DESCRIPTION
**Pull request by ybznek**
These scripts now use common syntax and using template_common.py. We can now modify behavior of all these scripts from 1 place - `template_common.py`.

I think, the script are now more readable and doesn't contains so much common code.
I have modified all relevant makefiles what I have found. But quite lot of them didn't work (before & after my changes)

### CODE REVIEW

File: shared/template_common.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    """
+    Call specified function on every line of file
+    """
+    with open(filename, 'r') as csv_file:
+        # put the CSV line's items into a list
+        lines = csv.reader(csv_file)
+        for line in lines:
+            if not line:
+                continue
+            if skip_comments and line[0].startswith('#'):
```
Comment (jan-cerny) [inline shared/template_common.py]: Are we sure the the `#` will be always at index 0 ?

The csv.reader doesn't seem to strip white characters. See:
```
with open("csv_ex","r") as f:
...   r = csv.reader(f)
...   for l in r:
...     print(l)
... 
['   Karel', '  Pepa', ' Petr      ', ' Milan', ' Michal']
```

Comment (ybznek) [inline shared/template_common.py]: I have took the code from previous version.

Comment (jan-cerny) [inline shared/template_common.py]: OK, then it is not necessary to fix that, because we already rely on that assumption.

**Comment by jan-cerny** [comment]
Overall it looks good to me. What Python versions did you use to test?

**Comment by ybznek** [comment]
It should use Python2. Python3 compatibility could be added later. But I prefer to do some deduplication, before.

**Comment by jan-cerny** [comment]
@ybznek OK,  I see. Thank you!

**Comment by iankko** [comment]
@jan-cerny 
> Are we sure the the # will be always at index 0 ?

The assumption here was if the sentence should be treated as comment, the '#' needs to be specified as the first one on the line. If somewhere else, it's probably desired content. Something like Docker is doing:
  https://docs.docker.com/engine/reference/builder/

exact part
```
Docker treats lines that begin with # as a comment, unless the line is a valid parser
directive. A # marker anywhere else in a line is treated as an argument. This allows
statements like:

# Comment
RUN echo 'we are running some # of cool things'
```

**Pull request merged**